src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -312,6 +315,8 @@ export default function PreferencesComponent() {
 					onChange={e => makeLocalChanges('defaultHelpUrl', e.target.value)}
 				/>
 			</div>
+
+
 			<Button
 				type='submit'
```

Comment:
I think it would be nice to have a discard button to undo changes. This may be outside the scope of this PR so it is okay to add this to the linked issue (or a new issue if needed).

Reply:
Made a new issue for this #1326